src/format_url.js

```diff
@@ -12,6 +12,7 @@ module.exports = function(path, accessToken) {
     }
 
     var url = (document.location.protocol === 'https:' || config.FORCE_HTTPS) ? config.HTTPS_URL : config.HTTP_URL;
+    url = (url.indexOf('/v4')) ? url.split('/v4')[0] : url;
```

Comment:
- Could this regex be a bit more permissive, e.g. in case anyone's pointing at v1-3?
- Are there any unit tests for this lib that could prove this code works?